tests/end_to_end/cucumber/features/steps/when.py

```diff
@@ -137,7 +162,7 @@ def run_antares_xpansion(context, method, memory=None, n: int = 1):
         context.lold = outputs.lold
         context.positive_unsupplied_energy = outputs.positive_unsupplied_energy
 
-
+#here we launch benderq directly
```

Comment:
The `when` description is pretty clear, useless comment (which does not match the following line by the way)